cfme/automate/explorer/method.py

```diff
@@ -108,7 +208,10 @@ def is_displayed(self):
 
 
 class Method(BaseEntity, Copiable):
-    def __init__(self, collection, name, display_name=None, location=None, script=None, data=None):
+    def __init__(self, collection, name=None, display_name=None, location='inline', script=None,
+                 data=None, cancel=False, validate=True, repository=None, playbook=None,
```

Comment:
**Optional**

Move cancel and validate args to the end of the list, after the object attribute args.